src/containers/org/OrgDataSetContainer.js

```diff
@@ -70,6 +97,12 @@ const OrgDataSetContainer = ({
     return contactString;
   }, [dataSet]);
 
+  useEffect(() => {
+    if (isValidUUID(dataSetId)) {
+      dispatch(getOrgDataSetSize({ dataSetId, organizationId }));
+    }
+  }, [dataSetId, dispatch, organizationId]);
```

Comment:
No change required, just a thought. This could also go inside `initializeOrganizationDataSet`.

Reply:
I like this idea. https://jira.openlattice.com/browse/APPS-2919